nntrainer/graph/network_graph.cpp

```diff
@@ -69,6 +70,21 @@ int NetworkGraph::compile(const std::string &loss_type) {
   return status;
 }
 
+void NetworkGraph::setExecutionOrder() {
+  auto node_count = graph.size();
+  /** @todo: remove backwarding count for non-trainble layers */
+  for (auto iter = cbegin(); iter != cend(); iter++) {
+    auto &node = *iter;
+    auto order_idx = iter - cbegin();
+    auto forward_order = order_idx;
+    auto calc_gradient_order = (node_count * 3) - order_idx + 1;
+    /** calc derivative is called right after calc_gradient */
```

Comment:
I'm not sure but. calc_gradient_order might be node_count + 2*(order_idx)+1

Reply:
Yes, you are right. I will update this.

Reply:
done.